### DISCUSSION TITLE
Will I be billed after the 1-year free trial of the API product expires?

### DISCUSSION THREAD

**Discussion by zlinoliver** [tuya/tuya-home-assistant#82] [Q&A]
After expiration, you can renew the subscription for free. The Trial Edition allows you to use all free API products but puts limits on the quota of API calls. For more information about the quota, see the [Pricing](https://developer.tuya.com/en/docs/iot/membership-service?id=K9m8k45jwvg9j).

**Comment by bh56** [comment]
It's not clear to me.  So you do have to renew the trial every 6 months.  Their pricing isn't very clear.  I rather not pay for connectivity.  Trying to decide if I should swap out of these devices to something more open

**Reply by Dvalin21** [reply]
They no longer have the 1 year free trail. Its 30 days and once its over they no longer allow you to renew the free trial. Because of this my devices no longer show up in my Home Assistant.

**Reply by bh56** [reply]
Thanks for the clarifying this for me.  Guess I might shy away from this platform of devices and only use these where I don't need integration to HASS.  Appreciate it.
